### PR TITLE
Fix url on whitepaper page to properly access whitepaper pdf [fixes #11918]

### LINKED ISSUE #11918: Bug report
### Describe the bug

A link to the original whitepaper is broken.

### To reproduce

1. Go to https://ethereum.org/en/whitepaper;
2. Click on [Researchers and academics seeking a historical or canonical version of the whitepaper [from December 2014] should use this PDF.(opens in a new tab)](https://ethereum.org/en/content/whitepaper/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf).

### Expected behavior

It should open the whitepaper in PDF format.

### Screenshots

image

### Desktop (please complete the following information)

- OS: Ubuntu - Browser: Brave - Version 1.61.116

### Smartphone (please complete the following information)

_No response_

### Additional context

_No response_

### Would you like to work on this issue?

- [ ] Yes
- [X] No

### PR DESCRIPTION
**Pull request by lucas-amberg**
## Description

Altered the URL on the whitepaper page to to access the PDF directly

## Related Issue

fixes #11918

### CODE REVIEW

**Comment by lucas-amberg** [comment]
This didn't fix it at all

**Comment by lucas-amberg** [comment]
I've spent a couple hours running around in circles trying to figure this out the best answer I've got is that its related to the locale URLs for the website, if you remove /en/ from the url you can access the PDF with no issue (https://ethereum.org/content/whitepaper/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf) vs (https://ethereum.org/en/content/whitepaper/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf). I'll maybe try again to fix this tomorrow but I can't figure it out at the moment

**Comment by lucas-amberg** [comment]
I'd also like to note that in the current dev branch of this repo this issue is non existent in the dev build (`yarn dev`) even with the locale path so I would assume that it has to do with the way Next.js is bundling the public folder when it is built for production

File: public/content/whitepaper/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ _This introductory paper was originally published in 2014 by Vitalik Buterin, th
 
 _While several years old, we maintain this paper because it continues to serve as a useful reference and an accurate representation of Ethereum and its vision. To learn about the latest developments of Ethereum, and how changes to the protocol are made, we recommend [this guide](/learn/)._
 
-[Researchers and academics seeking a historical or canonical version of the whitepaper [from December 2014] should use this PDF.](./whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf)
+[Researchers and academics seeking a historical or canonical version of the whitepaper [from December 2014] should use this PDF.](/content/whitepaper/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf)
```
Comment (TylerAPfledderer) [inline public/content/whitepaper/index.md:15]: For this to be properly fixed:
```suggestion
[Researchers and academics seeking a historical or canonical version of the whitepaper [from December 2014] should use this PDF.](/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf)
```

`/content/whitepaper` is already being added to the beginning through the `Link` component and an [isInternalPdf check](https://github.com/ethereum/ethereum-org-website/blob/0f4ae79ba3a8a4b00cf3d3bed422a43c954932ef/src/components/Link.tsx#L74)

**Review by TylerAPfledderer — CHANGES_REQUESTED** [review]
Hey @lucas-amberg!

This actually needed one tiny change, but the main issue still stands.

@pettinarip to amend the above comments made by @lucas-amberg: I believe this has to do with how NextJS is interpreting the URL. If I were to move the pdf file to the `assets` folder, change the link in question to `/assets/Ethereum_Whitepaper_-_Buterin_2014.pdf`, and remove the `isInternalPdf` check in the `Link` component then the loading of the file works as expected. 

Because we are using the `content` folder as an entry point for markdown files to be used as pages, Next is assuming the link in question is also just another page.

I do not have a solution at this time aside from moving the pdf to another folder like `assets`. Creating a redirect in a middleware file wasn't working when I tried it.

File: public/content/whitepaper/index.md (hunk 1 above)
Comment (wackerow) [reply public/content/whitepaper/index.md:15]: Hey @lucas-amberg! Thanks for the PR... I'll second @TylerAPfledderer... there is some logic in our `Link` component that helps with the paths here... Testing this locally, the above suggestion of just using `/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf` seems to do the trick. This approach should work even if `/en/` is in the path still.

Comment (TylerAPfledderer) [reply public/content/whitepaper/index.md:15]: @wackerow adding the language to the url is also breaking for me.

Comment (nhsz) [reply public/content/whitepaper/index.md:15]: @TylerAPfledderer @wackerow I'll submit a patch for this

**Comment by TylerAPfledderer** [comment]
@wackerow @pettinarip with the new change to the link the 404 still occurs here if a language code exists in the URL. Tried Arabic with `ar`.

I still think Next is interpreting the link as a page because of the language code existing in URL, and can't recognize it as an asset link. (In other words, there are no folders in the `public` directory that have the language codes as their name, so Next skips the asset check, however it does that!)

**Review by nhsz — APPROVED** [review]
LGTM

File: src/components/Link.tsx [hunk 2]
```diff
@@ -120,6 +118,32 @@ export const BaseLink = forwardRef(function Link(
     )
   }
 
+  if (isInternalPdf) {
+    return (
+      <ChakraLink
+        isExternal
```
Comment (nhsz) [inline src/components/Link.tsx:124]: this should be removed?

Comment (TylerAPfledderer) [reply src/components/Link.tsx:124]: The PDF should be opened in a separate tab, which means this is needed.

Comment (pettinarip) [reply src/components/Link.tsx:124]: Correct. We force it to open in a new tab.

**Review by nhsz — APPROVED** [review]

**Pull request merged**